src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -83,9 +83,10 @@ public void parseCommand_help() throws Exception {
     }
 
     @Test
-    public void parseCommand_list() throws Exception {
-        assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD) instanceof ListCommand);
-        assertTrue(parser.parseCommand(ListCommand.COMMAND_WORD + " 3") instanceof ListCommand);
+    public void parseCommand_viewProfiles() throws Exception {
+        String viewProfilesCommand = ViewProfilesCommand.COMMAND_WORD + " -" + ViewProfilesCommand.COMMAND_OPTION;
+        assertTrue(parser.parseCommand(viewProfilesCommand) instanceof ViewProfilesCommand);
+        assertTrue(parser.parseCommand(viewProfilesCommand + " 3") instanceof ViewProfilesCommand);
     }
```

Comment:
Perhaps it would be better to shift this test into `ProfileCommandParserTest` instead, since it's not exactly under the `AddressBookParser` anymore.